python/nav/statemon/RunQueue.py

```diff
@@ -197,7 +201,7 @@ def terminate(self):
         self.stop = 1
         self.await_work.notifyAll()
         self.lock.release()
-        debug("Waiting for threads to terminate...")
+        LOGGER.info("Waiting for threads to terminate...")
```

Comment:
Wow, I seriously think the old "debug" and "notice" levels of the statemon subsystem are both just plain old debug logging, nothing else :P